test-network-function/networking/suite_test.go

```diff
@@ -120,7 +125,15 @@ func TestListeningPortList(t *testing.T) {
 			expectedlisteningPorts: map[key]string{},
 		},
 	}
+	origFunc := utils.ExecuteCommand
+	defer func() {
+		utils.ExecuteCommand = origFunc
+	}()
 	for _, tc := range testCases {
+		utils.ExecuteCommand = func(command string, timeout time.Duration, context *interactive.Context) (string, error) {
+			output, err := os.ReadFile(tc.jsonFileName)
```

Comment:
This UT is still failing. You have to initialize the tc.jsonFileName for each TC (or make it a global variable if all TCs can use the same file) on this unit test as you did on TestDeclaredPortList.

Reply:
is works now.